pkgs/development/libraries/concord/default.nix

```diff
@@ -1,8 +1,4 @@
-{ stdenv
-, lib
-, fetchFromGitHub
-, curl
-}:
+{ stdenv, lib, fetchFromGitHub, curl }:
```

Comment:
Please keep them on seperate lines, makes future additions/removals/changes easier to diff

Reply:
👆